### PR TITLE
2021q3 Git entry with input from imp and lwhsu

### PR DESCRIPTION
**Pull request by Jehops**

### CODE REVIEW

File: 2021q3/git.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+link:http://gameoftrees.org/[Game of Trees] URL: link:http://gameoftrees.org/[http://gameoftrees.org/] +
+link:https://github.com/johnmehr/gitup[gitup] URL: link:https://github.com/johnmehr/gitup[https://github.com/johnmehr/gitup]
+
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org> +
+Contact: Warner Losh <imp@FreeBSD.org> +
+Contact: Ed Maste <emaste@FreeBSD.org> +
+Contact: link:https://lists.freebsd.org/mailman/listinfo/freebsd-git[FreeBSD-git mailing list] +
+Contact: #gitcvt channel on EFnet
+
+With the end the third quarter of 2021, we are approaching the one-year anniversary of the transition of the doc and src repositories from Subversion to Git.  The 2021Q4 branch of the ports tree has been created, the third such quarterly branch created using Git.
```
Comment (pauamma) [inline 2021q3/git.adoc:20]: s/end the/end of the/

File: 2021q3/git.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+link:https://www.freshports.org/net/gitup[Gitup] and link:https://www.freshports.org/devel/got[Game of Trees] are currently available as ports and packages.
+Future work will evaluate them as candidates for the base system.
+
+Remaining work includes continuing with refinements to Git documentation in the Handbook, committing new and updated repository hooks for managing branch content and commit mail, and surveying pre-commit hooks to use the CI cluster to build release artifacts.  Priorities have been set for the next working group tasked with refining our Git workflow.  The first meeting will begin in mid-October.
```
Comment (pauamma) [inline 2021q3/git.adoc:27]: s/will begin/will be/

**Review by pauamma — CHANGES_REQUESTED** [review]

**Review by pauamma — APPROVED** [review]
LGTM now.

**Comment by Jehops** [comment]
No, you're not missing anything. I didn't mean to click the link.

**Pull request merged**